release_notes.py

```diff
@@ -276,6 +276,15 @@ def createFileReleaseNotes(fileName, deleteFilePath):
         names = fileName.split("\t")
         changeType = names[0]
         fullFileName = names[1]
+
```

Comment:
@aviadl @orius123 

When I renamed `download_secret_conf_file.sh` to `download_demisto_conf.sh` - this script failed
python could not open the `.sh` file in `r` mode

I refactor this a bit to skip anything that doesn't have `fileTypeMapping` (anything that is not script/integration/playbook/report